identity/identity.go

```diff
@@ -461,3 +460,93 @@ func (i *Identity) WithDeclassifiedCredentialsOIDC(ctx context.Context, c cipher
 	ii.Credentials = credsToPublish
 	return &ii, nil
 }
+
+// Patch Identities Parameters
+//
+// swagger:parameters patchIdentities
+//
+//nolint:deadcode,unused
+//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
+type patchIdentitites struct {
+	// in: body
+	Body PatchIdentitiesBody
+}
+
+// Patch Identities Body
+//
+// swagger:model patchIdentitiesBody
+//
+//lint:ignore U1000 Used to generate Swagger and OpenAPI definitions
+type PatchIdentitiesBody struct {
+	// Identities holds the list of patches to apply
+	//
+	// required
+	Identities []*IdentityPatch `json:"identities"`
+
+	// Future fields:
+	// RemotePatchesURL string
+	// Async bool
+}
+
+// Payload for patching an identity
+//
+// swagger:model identityPatch
```

Comment:
This is a bit misleading, because we also support patching identities individually using JSON Patch. Maybe `batchIdentityPatch`?